sdk/cognitiveservices/ms-azure-cs-inkrecognizer/inkrecognizer/build.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.2"
+    defaultConfig {
+        minSdkVersion 23
```

Comment:
The min API level is set to 23. All Java 8 features are not supported in 23, e.g. Reactor (the dependency specified below), java.time won't work on devices with API Level 23. The min level should be 26 to get all Java 8 features & for reactor.

Reply:
Thanks for the comment, I have changed the version to 26.